src/main/java/marquez/MarquezApp.java

```diff
@@ -15,16 +15,22 @@
 import marquez.resources.JobResource;
 import marquez.resources.OwnerResource;
 import marquez.resources.PingResource;
+import org.flywaydb.core.Flyway;
+import org.flywaydb.core.api.FlywayException;
 import org.jdbi.v3.core.Jdbi;
 import org.jdbi.v3.postgres.PostgresPlugin;
 import org.jdbi.v3.sqlobject.SqlObjectPlugin;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MarquezApp extends Application<MarquezConfig> {
```

Comment:
why the rename to `MarquezApp`? Just for convenience?

Reply:
Yeah, partially for convenience and also just a habit of mine to abbreviate when possible. (I initially referenced the dropwizard docs to get started).